src/current/_includes/releases/v24.2/v24.2.10.md

```diff
@@ -0,0 +1,55 @@
+## v24.2.10
+
+Release Date: February 6, 2025
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-2-10-general-changes">General changes</h3>
+
+- The protected timestamp (PTS) records of running changefeeds are now updated when the set of targets changes, such as when system tables are added to the protected tables list. [cockroachdb/cockroach#138653][#138653]
+
+<h3 id="v24-2-10-sql-language-changes">SQL language changes</h3>
+
+- The `legacy_varchar_typing` session setting has been added, which reverts the changes of #133037 that cause the change in typing behavior described in #137837. Specifically, it makes type-checking and overload resolution ignore the newly added "unpreferred" overloads. This setting defaults to `on`. [#138002][#138002]
```

Comment:
I'd link the PR #133037 in this case, since it won't be intuitive without a link.

Reply:
added the links!